apps/api-extractor/src/collector/Collector.ts

```diff
@@ -158,6 +182,17 @@ export class Collector {
     return this._dtsLibReferenceDirectives;
   }
 
+  /**
+   * A list of names (e.g. "runtime-library") that should appear in a path-based reference like this:
+   *
+   * ```
+   * /// <reference path="runtime-library" />
+   * ```
+   */
+  public get dtsFileReferenceDirectives(): ReadonlySet<string> {
```

Comment:
The `runtime-library` example is not intended to be a path.  Maybe something like this:
```ts
  /**
   * A list of paths (e.g. "path/to/file.d.ts") that should appear in a reference like this:
   *
   * ```
   * /// <reference path="path/to/file.d.ts" />
   * ```
   *
   * The paths are resolved relative to the source file containing the reference.
   */
```

Reply:
ah yeah just missed this when copying the doc string :+1: